tutorials/tts/FastPitch_Training_GermanTTS.ipynb

```diff
@@ -0,0 +1,1226 @@
+    "  1. **Introduction**: FastPitch and HiFi-GAN in NeMo\n",
+    "  2. **Dataset Preparation**: How to prepare German dataset for FastPitch\n",
+    "  3. **Training**: Example of FastPitch training and evaluation\n",
+    "  4. **Finetuning HiFi-GAN**: Improving speech quality by Finetuning HiFi-GAN on synthesized mel-spectrograms from FastPitch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f6e7115",
+   "metadata": {},
+   "source": [
+    "# License\n",
+    "\n",
+    "> Copyright 2022 NVIDIA. All Rights Reserved.\n",
+    "> \n",
+    "> Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "> you may not use this file except in compliance with the License.\n",
+    "> You may obtain a copy of the License at\n",
+    "> \n",
+    ">     http://www.apache.org/licenses/LICENSE-2.0\n",
+    "> \n",
+    "> Unless required by applicable law or agreed to in writing, software\n",
+    "> distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "> WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "> See the License for the specific language governing permissions and\n",
+    "> limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a900ecb7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can either run this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies# .\n",
+    "\"\"\"\n",
+    "BRANCH = 'r1.10.0'\n",
+    "# # If you're using Colab and not running locally, uncomment and run this cell.\n",
+    "# !apt-get install sox libsndfile1 ffmpeg\n",
+    "# !pip install wget unidecode pynini==2.1.4 scipy==1.7.3\n",
+    "# !python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[all]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c9748fb2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import nemo\n",
+    "import torch\n",
+    "import librosa\n",
+    "import numpy as np\n",
+    "\n",
+    "from pathlib import Path\n",
+    "from tqdm.notebook import tqdm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b8813aef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# lets download the files we need to run this tutorial\n",
+    "\n",
+    "!mkdir NeMoGermanTTS\n",
+    "!cd /NeMoGermanTTS && wget https://raw.githubusercontent.com/nvidia/NeMo/$BRANCH/scripts/dataset_processing/tts/openslr/get_data.py\n",
```

Comment:
Would be nice if repeated values like "NeMoGermanTTS" could be changed to a constant/variable